frontend/src/reducer/notificationPage.js

```diff
@@ -74,6 +74,13 @@ export default function notificationPage (state = defaultNotificationsObject, ac
       }
     }
 
+    case `${READ}/${NOTIFICATION_LIST}`: {
+      const notificationList = action.notificationList.map(notification => (
```

Comment:
In order to have a more stable action we should use the state here and not action data, because if it use action data (even if it will exactly the same data because we use it correctly) it doesn't ensure that the notificationList is exactly the same with state.notificationList

Reply:
You are right, we don't even need to pass the action data. I'm gonna change it.